src/nidigital/examples/nidigital_ppmu_source_and_measure.py

```diff
@@ -0,0 +1,187 @@
+#!/usr/bin/python
+
+import argparse
+import nidigital
+import os
+import pytest
+import sys
+import time
+
+
+def example(resource_name, options, channels, measure, aperture_time,
+            command, settling_time=None, current_level_range=None, current_level=None,
```

Comment:
I think you can set default value of `command` to `None`, which is what will be passed to this method if the user doesn't specify any sub-command.

Reply:
Done